### PR TITLE
[LICENSE] Introduce Apache Rat

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #432](https://github.com/yaooqinn/kyuubi/pull/432)    Target Issue [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
How abount 
```xml
      <plugin>
        <groupId>org.apache.rat</groupId>
        <artifactId>apache-rat-plugin</artifactId>
        <inherited>true</inherited>
        <configuration>
          <excludes>
            <exclude>.git/</exclude>
            <exclude>.github/**</exclude>
            <exclude>.gitignore</exclude>
            <exclude>.gitmodules</exclude>
            <exclude>.editorconfig</exclude>
            <exclude>LICENSE</exclude>
            <......
          </excludes>
        </configuration>
      </plugin>
```

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -55,6 +55,8 @@ jobs:
         run: |
           mvn clean install ${{ matrix.profiles }} -Dmaven.javadoc.skip=true -V
           bash <(curl -s https://codecov.io/bash)
+      - name: Check License
```
Comment (yaooqinn) [inline .github/workflows/master.yml:58]:
```yaml  
rat:
    name: Check License
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-java@v1
        with:
          java-version: '1.8'
      - run: mvn org.apache.rat:apache-rat-plugin:check
```

mv this to a single job

**Comment by pan3793** [comment]
OK, I will move to rat maven plugin

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.2.0
